rust/tests/proxy_tests.rs

```diff
@@ -19,16 +19,19 @@ use assert_cmd::Command;
 
 use exitcode::DATAERR;
 
-use wiremock::MockServer;
-
 #[tokio::test]
-async fn ok_proxy_test() {
```

Comment:
Why not have both ok and wrong tests?

Reply:
I removed that test in a second commit since I discovered that test was actually not working. When it passed, it did since the driver was cached, but the mocked proxy was not actually doing anything. In fact, if the flag `--clear-cache` is added the test, the test always fails. I looked for an alternative mocked proxy in Rust but I haven't found anything. So I decided to drop the test. I added an update in the original PR description to inform about it.